R/execution.r

```diff
@@ -237,6 +237,11 @@ execute = function(request) {
             if (!plot_builds_upon(last_recorded_plot, plotobj)) {
                 send_plot(last_recorded_plot)
             }
+            # need to be set here to capture the size and have it available
+            # when the plot is sent
+            # 7 is the default in repr...
+            attr(plotobj, '.irkernel_width') <- getOption("repr.plot.width", 7)
+            attr(plotobj, '.irkernel_height') <- getOption("repr.plot.height", 7)
```

Comment:
i don’t like code repeating itself.

IRkernel depends on repr, and repr sets the option on load, so the option will be set unless explicitly unset by an user. please remove the defaults from the getOption calls and the “7 is the default in repr...” comment